README.md

```diff
@@ -120,3 +120,39 @@ Usage
 -----
 
 To be written.
+
+Spatial Function Reference
+--------------------------
+
+This is a list of all functions which are are currently implemented in the geo project. Some functions are only available
+if you use a specific geometry engine.
+
+| Function Name | GEOS | PostGIS | MySQL | SpatiaLite | OpenGIS standard |
+|---------------|------|---------|-------|------------|------------------|
+|`area`         |  X   |    X    |   X   |     X      |        X         |
+|`boundary`     |  X   |    X    |       |     X      |        X         |
+|`buffer`       |  X   |    X    |       |     X      |        X         |
```

Comment:
There is a function `buffer` but not a function `ST_Buffer`. We would have to differentiate between PostGIS and MySQL then.